### PR TITLE
Display hosted URL in CLI while launching kedro viz

### LINKED ISSUE #1638: Display URL in CLI after launching `kedro viz`
## Description
As per title.

## Context
If for some reason the user closes the tab or is on a headless environment (without browser), it's not clear at all what to do:

```
$ kedro viz
Starting Kedro Viz Backend Server...
Kedro Viz Backend Server started successfully...

# empty space
```

## Possible Implementation
```
$ kedro viz
Starting Kedro Viz Backend Server...
Kedro Viz Backend Server started successfully...

✨ Kedro Viz is running at
http://127.0.0.1:1414/
```

## Possible Alternatives
?

## Checklist

- [ ] Include labels so that we can categorise your feature request

### PR DESCRIPTION
**Pull request by ravi-kumar-pilla**
## Description

Resolves #1638 

## Development notes

* Added print statement to display the url on which kedro viz has started running
* Fixed dependency conflict of s3fs with kedro-datasets

## QA notes
* Install package using `pip install -e package`
* `cd demo-project`
* Run `kedro viz`
* Url information as below should be displayed

image

## Checklist

- [x] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [x] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

### CODE REVIEW

**Comment by astrojuanlu** [comment]
I tested this with `make run` and I don't see the URL, is that expected?

```
  ~/Projects/QuantumBlack Labs/kedro-viz  feature/display-cli-url !1 ····  29s 23:41:04
❯ make run                                                               (kedro-viz38-dev) 
PYTHONWARNINGS="ignore:Kedro is not yet fully compatible" PYTHONPATH="/Users/juan_cano/Projects/QuantumBlack Labs/kedro-viz/package" python3 package/kedro_viz/server.py demo-project
Starting Kedro Viz Backend Server...
[11/19/23 23:41:10] WARNING  'exists()' not implemented for 'ImageDataSet'.     core.py:299
                             Assuming output does not exist.                               
                    WARNING  'exists()' not implemented for 'ImageDataSet'.     core.py:299
                             Assuming output does not exist.                               
[11/19/23 23:41:15] INFO     Checking for update...                          __init__.py:18
```

File: package/kedro_viz/launchers/cli.py [hunk 1]
```diff
@@ -165,7 +165,10 @@ def viz(
 
         _wait_for(func=_check_viz_up, host=host, port=port)
 
-        print("Kedro Viz Backend Server started successfully...")
+        print(
+            "Kedro Viz Backend Server started successfully. \n\n"
```
Comment (rashidakanchwala) [inline package/kedro_viz/launchers/cli.py:169]:
```suggestion
            "Kedro Viz started successfully. \n\n"
```

**Comment by ravi-kumar-pilla** [comment]
> I tested this with `make run` and I don't see the URL, is that expected?
> 
> ```
>   ~/Projects/QuantumBlack Labs/kedro-viz  feature/display-cli-url !1 ····  29s 23:41:04
> ❯ make run                                                               (kedro-viz38-dev) 
> PYTHONWARNINGS="ignore:Kedro is not yet fully compatible" PYTHONPATH="/Users/juan_cano/Projects/QuantumBlack Labs/kedro-viz/package" python3 package/kedro_viz/server.py demo-project
> Starting Kedro Viz Backend Server...
> [11/19/23 23:41:10] WARNING  'exists()' not implemented for 'ImageDataSet'.     core.py:299
>                              Assuming output does not exist.                               
>                     WARNING  'exists()' not implemented for 'ImageDataSet'.     core.py:299
>                              Assuming output does not exist.                               
> [11/19/23 23:41:15] INFO     Checking for update...                          __init__.py:18
> ```

I thought of fixing only the `kedro viz` command log statements and `make run` uses a different route to start the server. But it makes sense to fix both in this ticket. I have added the log statements to make run and made `make run`, `kedro viz` to follow the similar start pattern. Thank you !

**Review by rashidakanchwala — APPROVED** [review]
LGTM! thanks

**Review by NeroOkwa — APPROVED** [review]
LGTM!

I tested this locally with ```kedro viz``` and it worked perfectly.

**Pull request merged**